openedx/core/djangoapps/discussions/models.py

```diff
@@ -147,6 +150,14 @@ class Features(Enum):
         )._asdict(),
     },
     'discourse': {
+        'additional_messages': [
+            _(
+                '{provider} requires that LTI advanced sharing be enabled for your course'
+                ' , as this provider uses email address and username to personalize'
+                ' the experience. Please contact {support_contact} to enable this feature.'
+            ).format(
+                provider='Discourse', support_contact=_('technical support'))
```

Comment:
The issue description mentions `support@edx.org` here but I'm guessing this should be configurable?

Reply:
I see the following used as teh support email in some places: 
configuration_helpers.get_value('CONTACT_EMAIL', settings.CONTACT_EMAIL)

This value is dynamic, and could be different per-site, so it either needs to be filled in dynamically, or need to use the global setting.